pages.hi/common/chsh.md

```diff
@@ -0,0 +1,7 @@
+# chsh
+
+> उपयोगकर्ता का लॉगिन शेल बदलें।
+
+- शेल बदलें:
+
+`chsh -s {{path/to/shell_binary}} {{username}}`
```

Comment:
Please translate this arguments for non-English speakers.

Reply:
Updated, please check.